cli/pkg/kctrl/cmd/package/release/release.go

```diff
@@ -159,6 +153,18 @@ func (o *ReleaseOptions) releaseResources(appSpec kcv1alpha1.AppSpec, pkgBuild c
 	return nil
 }
 
+func generateValuesSchema(pkgBuild cmdpkgbuild.PackageBuild) (*kcdatav1alpha1.ValuesSchema, error) {
+	var yttPaths []string
+	for _, templateStage := range pkgBuild.Spec.Template.Spec.App.Spec.Template {
+		if templateStage.HelmTemplate != nil {
+			return NewHelmValuesSchemaGen(templateStage.HelmTemplate.Path).Schema()
+		} else if templateStage.Ytt != nil {
+			yttPaths = append(yttPaths, templateStage.Ytt.Paths...)
```

Comment:
If the values are fed to only the first templating step even if it is YTT
Maybe we should stop doing this

Reply:
Not able to understanfd. But looks like here, you are trying to collate all the folder paths for ytt.